### ISSUE TITLE
Drop new Function constructs

### ISSUE THREAD

**Issue by dgerber** [jstat/jstat#125]
Hi,
In node.js repl:
```
> require('jStat').jStat().rand(3)
ReferenceError: jStat is not defined
```
because  of the unusual scope resulting from the Function constructor.

In core.js:455:
```diff
<     jProto[passfunc] = function() {
<       return jStat(jStat[passfunc].apply(null, arguments))
<     };
---
>     jProto[passfunc] = new Function(
>         'return jStat(jStat.' + passfunc + '.apply(null, arguments));');

```

**Comment by trevnorris** [comment]
Thanks. Will take care of it.

**Comment by trevnorris** [comment]
This was fixed in 07157cc5.
